proposals/notifications.md

```diff
@@ -0,0 +1,155 @@
+# Solid Notifications
+
+Solid notifications are intended to announce social interactions and data. A
+Solid notification is an [LDP Resource](https://www.w3.org/TR/ldp/#ldpr) which
+is submitted to and resides in an [LDP
+Container](https://www.w3.org/TR/ldp/#ldpc), i.e., a [Solid inbox](inbox.md).
+
+The contents of a notification is application specific. We derive two core
+principles from this:
+
+1. a notification can contain arbitrary data
+2. a notification is vocabulary agnostic
+
+Applications can create notifications to use any vocabulary to describe its
+contents in any shape and form. Multiple notifications can be contained inside a
+single resource or span across different resources. A single resource with
+multiple notifications may be suitable for `HTTP PATCH`, whereas multiple
+resources containing a single notification may be suitable for `HTTP POST` or
+`PUT`. If different access controls are desired for inboxes and notifications,
+implementing a multiple notification approach with `HTTP POST` or `PUT` may be
+preferable.
+
+
+## Namespaces and document conventions
+The following namespaces are used in this document:
+
+|Prefix|Namespace
+|------|---------
+|solid|http://www.w3.org/ns/solid/terms#
+|xsd|http://www.w3.org/2001/XMLSchema#
+|dcterms|http://purl.org/dc/terms/
+|sioc|http://rdfs.org/sioc/ns#
+|foaf|http://xmlns.com/foaf/0.1/
+|pingback|http://purl.org/net/pingback/
+|as|http://www.w3.org/ns/activitystreams#
+|schema|https://schema.org/
+
+All RDF examples are written in Turtle syntax.
+
+
+## Discovery
+
+An inbox container can be discovered via the `solid:inbox` property, in which
+each notification can be discovered via `ldp:contains` relations. Notifications
+that are of LDP RDF Source should be instances of `solid:Notification`. Although
+`solid:Notification` is not required to discover notifications, it is good
```

Comment:
Are we sure about this part? If we don't require apps to use `solid:Notification` property, aren't we basically saying "anything in the inbox is potentially a notification, figure it out yourself"?